exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,8 @@
 // Start by creating a variable `message`
+var greetingStart = "My name is ";
+var greetingMiddle = " and my name is ";
+var greetingEnd = " characters long";
+var myName = "Simon";
+var nameLength = myName.length;
 
-console.log(message);
+console.log(greetingStart + myName + greetingMiddle + nameLength + greetingEnd);
```

Comment:
Could also use string interpolation
```suggestion
console.log(`My name is ${myName} and my name is ${nameLength} characters long`);
```